clusterloader2/testing/batch/config.yaml

```diff
@@ -1,12 +1,22 @@
+{{$mode := (DefaultParam .MODE "Indexed")}}
+{{$pods_per_node_per_size := 20}}
+{{$total_pods_per_size := MultiplyInt .Nodes $pods_per_node_per_size}}
+{{$small_job_size := 5}}
+{{$small_jobs_count := DivideInt $total_pods_per_size $small_job_size}}
+{{$medium_job_size := 20}}
+{{$medium_jobs_count := DivideInt $total_pods_per_size $medium_job_size}}
+{{$large_job_size := 400}}
+{{$large_jobs_count := DivideInt $total_pods_per_size $large_job_size}}
+
 name: batch
 
 namespace:
   number: 1
 
 tuningSets:
-- name: Uniform1qps
+- name: Uniform5qps
   qpsLoad:
-    qps: 1
+    qps: 5
```

Comment:
Are you sure you want to use so low constant value?

For CL2 test we are using `--env=CL2_LOAD_TEST_THROUGHPUT=50` to determine QPS in saturation tunning test.